fee-monitor/src/index.ts

```diff
@@ -183,7 +183,14 @@ async function main() {
     for (;;) {
         console.log();
         console.log(`BlockNumber: ${blockNumber}`);
+        const cloneChecker = (instance: DynamicChecker): DynamicChecker => {
+            const copy = new (instance.constructor as { new (): DynamicChecker })();
+            Object.assign(copy, instance);
```

Comment:
As far as I know, there is no easy way to deep copy a JavaScript object.
Please create a manual copy function.

This repl used the `Object.assign` but the array type is shared between the copied and original.
https://repl.it/@majecty/Objectassign

Reply:
OMG, I just believed the assign simulates deep copies : ( I'll manually implement deep copy for Dynamic checker.

Reply:
lodash's cloneDeep resolves this issue. I utilized it.

Reply:
Since we are using class, lodash's cloneDeep will not clone perfectly.
https://stackoverflow.com/a/34295179/2756490

Reply:
Hmm... It will clone the data recursively and I just utilized the typescript type rather than `instanceof` check. Is it problematic for the cloned class not to have an inappropriate prototype?

Reply:
If the code is not using `instanceof`, it's okay. Please write a comment that the `deepClone` will change the result of `instanceof`.

Reply:
Ok, I'll add a comment.